packages/forms-web-app/src/validators/common/schemas/file-upload.js

```diff
@@ -33,32 +33,39 @@ const schema = (noFilesError) => ({
         }
 
         const uploadedFiles = !Array.isArray(files[path]) ? [files[path]] : files[path];
+        const isSingleFile = uploadedFiles.length === 1;
+        const singleFileMimeTypeErrorMsg =
+          'The selected file must be a pdf, doc, docx, tif, tiff, jpg, jpeg or png';
+        const multiFileMimeTypeErrorMsg = 'must be a DOC, DOCX, PDF, TIF, JPG or PNG';
 
         uploadedFiles.forEach(({ mimetype, name }) => {
-          validMimeType(
-            mimetype,
-            Object.values(mimeTypes),
-            `${name} must be a DOC, DOCX, PDF, TIF, JPG or PNG`
-          );
+          const errorMsg = isSingleFile
+            ? singleFileMimeTypeErrorMsg
+            : `${name} ${multiFileMimeTypeErrorMsg}`;
+          validMimeType(mimetype, Object.values(mimeTypes), errorMsg);
         });
 
-        const { name } = req.files[path];
-        await validAV(req.files['file-upload'], name);
-
         await Promise.all(
           uploadedFiles.map((file) =>
             validateMimeBinaryType(
               file,
               Object.values(mimeTypes),
-              `${file.name} must be a DOC, DOCX, PDF, TIF, JPG or PNG`
+              isSingleFile
+                ? singleFileMimeTypeErrorMsg
+                : `${file.name} ${multiFileMimeTypeErrorMsg}`
             )
           )
         );
 
-        uploadedFiles.forEach(({ size, fileName }) => {
-          validateFileSize(size, uploadApplicationMaxFileSize, fileName);
+        const sizeErrorMsg = isSingleFile ? 'The selected file must be smaller than 15MB' : null;
+
+        uploadedFiles.forEach(({ size, name }) => {
+          validateFileSize(size, uploadApplicationMaxFileSize, name, sizeErrorMsg);
         });
 
```

Comment:
@guiribeirokc I have moved clamav to after the validateFileSize check - otherwise clamav will return its own error message for an over-size file is not in the correct format (i.e it will bypass validateFileSize).